### PR TITLE
License documentation updates

### PR DESCRIPTION
**Pull request by apg-dev**
Miscellaneous updates to the license agreements.

### CODE REVIEW

File: en/Advanced topics/Catalyst license.md [hunk 1]
```diff
@@ -1,13 +1,13 @@
-Catalyst license is an optional license to use Obsidian under. It has three tiers ($25, $50, $100), all of them give you [[Insider builds|access to insider builds]].
+Obsidian is free for personal use. A Catalyst license is an optional license that provides the following benefits:
 
-Please note that Catalyst is optional and ==does not give you any additional feature==.
+1. It gives you early access to the latest, cutting edge versions of the app
```
Comment (ericaxu) [inline en/Advanced topics/Catalyst license.md:3]: Would appreciate some punctuation at the end of each list item, but not sure if comma, period, or semicolon works the best.

File: en/Advanced topics/Commercial license.md [hunk 2]
```diff
@@ -1,53 +1,53 @@
 ### About Obsidian commercial license
 
-Obsidian is free for personal use. For commercial use in a company of at least 2 people, you need to purchase a commercial license in order to use it for work-related activities such as taking meeting notes or keeping track of a team project.
+Obsidian is free for personal use. For commercial (business) use in a company of two (2) or more employees, a commercial license is required. Business use includes any work-related activities such as capturing meeting notes, maintaining team project notes, configuring an organizational wiki, and similar note taking / document management / knowledge management functionality.
```
Comment (ericaxu) [inline en/Advanced topics/Commercial license.md:3]: Thanks for making it more consistent with the EULA to make things clearer!

Nit: I would prefer no space before and after the slash.

File: en/Advanced topics/Commercial license.md [hunk 3]
```diff
@@ -1,53 +1,53 @@
...
 
 > *Q1. I'm a freelancer/writer/blogger, do I need a commercial license?*
 > \
-> A: Unless you hire at least one person full time, you do not need a commercial license.
+> A: Unless you hire at least one additional person, you do not need a commercial license.
 
 > *Q2. I'm a teacher/professor and I use Obsidian for my teaching/lectures, do I need a commercial license?*
 > \
-> A: No, if it's related to education in general, even if you work in a for-profit institution, you do not need a commercial license.
+> A: Obsidian usage for education-related activities within an .edu organization does not require a commercial license.
```
Comment (ericaxu) [inline en/Advanced topics/Commercial license.md:39]: Could you check ".edu organization" to something more generic? Canadian universities don't end with .edu, for example. Maybe schools and educational institution?

**Review by ericaxu — CHANGES_REQUESTED** [review]
Left some comments, thanks so much for proofreading and editing!

File: en/Advanced topics/Catalyst license.md (hunk 1 above)
Comment (apg-dev) [reply en/Advanced topics/Catalyst license.md:3]: Np. I'll add semicolons.

File: en/Advanced topics/Commercial license.md (hunk 2 above)
Comment (apg-dev) [reply en/Advanced topics/Commercial license.md:3]: The problem with removing the spaces is that you get weird line spacing, such as below. I recommend leaving in the spaces or changing slashes to commas. Let me try commas and see how that looks.

image

File: en/Advanced topics/Commercial license.md (hunk 3 above)
Comment (apg-dev) [reply en/Advanced topics/Commercial license.md:39]: Good catch. I'll add "schools and other recognized educational institutions."

**Review by ericaxu — APPROVED** [review]

**Comment by ericaxu** [comment]
Looks great, thanks so much @apg-dev !

**Pull request merged**